site-src/blog/2023/0823-welcome-gamma.md

```diff
@@ -0,0 +1,160 @@
+---
+description: >
+  We are excited to announce the v0.8.0 release of Gateway API, where the GAMMA
+  initiative has now reached Experimental status, Gateway and HTTPRoute gain
+  some new capabilities, conformance profiles are supported, and more!
+---
+
+# Gateway API: Welcome, GAMMA!
+
+<small style="position:relative; top:-30px;">
+  :octicons-calendar-24: August 09, 2023 ·
+  :octicons-clock-24: 5 min read
+</small>
+
+We are thrilled to announce the v0.8.0 release of the Gateway API! With this
```

Comment:
Comment applies throughout, but generally prefer "Gateway API" instead of "The Gateway API"

```suggestion
We are thrilled to announce the v0.8.0 release of Gateway API! With this
```